source/MaterialXTest/MaterialXGenShader/GenShader.cpp

```diff
@@ -98,116 +98,6 @@ TEST_CASE("GenShader: TypeDesc Check", "[genshader]")
     REQUIRE_THROWS(mx::TypeDesc::get("bar"));
 }
 
-TEST_CASE("GenShader: OSL Reference Implementation Check", "[genshader]")
```

Comment:
Remove this test as the legacy OSL implementations are outdated and are going to be removed anyway, and so I didn't want to spend time on updating this.